pkg/cmd/workload/csv_server.go

```diff
@@ -49,6 +50,7 @@ func runCSVServer(_ *cobra.Command, _ []string) error {
 			}
 		})
 	}
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
```

Comment:
I just added this one, should I do all of them?

Reply:
Might as well add the memory one as well. The others don't seem as important.